tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -191,6 +186,16 @@ func (s *stresser) Stress() error {
 	}
 	s.stressTable = createStressTable(stressEntries)
 
+	return nil
+}
+
+func (s *stresser) Stress() error {
+	if err := s.setup(); err != nil {
```

Comment:
please fold the code back into `Stress`? `setup()` is just as much setup code the code launching the goroutines; there's no meaningful distinction based on the name alone